src/datasource-zabbix/datasource.js

```diff
@@ -640,6 +640,15 @@ class ZabbixAPIDatasource {
     );
     return useTrends;
   }
+
+  setDescription(items, options) {
+    let panelModel = this.dashboardSrv.dash.getPanelById(options.panelId);
+    let desc = items.map(i => i.description);
+    desc = desc.length === 1 ? desc[0] : desc;
```

Comment:
Cant understand the logic of this. `desc` is array, why should we assign it to variable's value?

Reply:
Array will produce string: `[ item ]` but single value just `item`. So, when there is only one item I decided to unwrap this item from array.